src/plugins/intel_cpu/docs/fusing_disabling.md

```diff
@@ -0,0 +1,53 @@
+# Fusing disabling
+Fusing disabling is controlled by environment variables.
+
+The variables define conditions of the node which fusing should be disabled for.
+
+Fusing can be disabled for:
+
+- target node (node which we fuse into):
+
+  ```sh
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='>Convolution Pooling'
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='Convolution Pooling'
+  ```
+
+- fused node (node which we fuse):
+
+  ```sh
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='<Eltwise FakeQuantize'
```

Comment:
The same. Would be nice to have a comment,

Reply:
Will add few words

Reply:
Done